src/lib/utils/pdf-generator.ts

```
@@ -14,244 +14,301 @@ export async function downloadPDF(article: Article) {
+				currentY = margin;
+			}
 
-							// Check if image needs a new page
-							if (currentY + imgHeight > pageHeight - margin) {
-								pdf.addPage();
-								currentY = margin;
-							}
+			// Handle different content types
+			if (section instanceof HTMLHeadingElement) {
+				pdf.setFont('helvetica', 'bold');
+				let fontSize, marginTop, marginBottom;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)